Sources/SourceKitLSP/SourceKitLSPServer.swift

```diff
@@ -2472,14 +2472,28 @@ extension IndexSymbolKind {
   }
 }
 
+fileprivate extension IndexSymbolKind {
+  /// Whether this symbol should be considered as a container in `SymbolOccurrence.containerName`
+  var isContainer: Bool {
```

Comment:
Feels like this should have something to do with type in its name (though also includes module/namespace...) given it doesn't include functions. Same with `containerName`.

Also `union` should probably be included in the `true` case either way.

Reply:
Would probably be best to avoid using the word `container` too, to avoid confusion with the `containedBy` relation. Maybe something like `hasChildren`?

Reply:
Really this is just: Should this be considered for `containerName` and I think it’s actually easier to just have the `switch` inside the `filter` of `containerName`.